ichnaea/webapp/tests.py

```diff
@@ -113,18 +111,23 @@ def test_redis(self, broken_app):
         assert res.json["redis"] == {"up": False, "time": 0}
 
     def test_lbheartbeat(self, broken_app):
+        """LBHeartbeat returns OK for a broken application."""
```

Comment:
I added this comment because my first instinct was to delete this test ("Why is ``__lbheartbeat__`` tested in the ``__heartbeat__`` tests?). The purpose of the test is to show that ``__lbheartbeat__`` returns "healthy" even for a mis-configured application. If it did not, AWS may stop sending traffic, and a auto-scaling group may reboot the web head, making it more difficult to diagnose the root issue.